schemas/documentation_generation.py

```diff
@@ -54,26 +60,40 @@ class FirestoreDoc(BaseModel):
     owner: Optional[str] = None
 
 
+class IdentifiedItemToDocument(BaseModel):
```

Comment:
Should we add a field with Type? So the UI can tell between a folder and a file. This way we can display that UI feature we talked about:
`You are about to document 100 files and 20 folders`

Reply:
Yes the only thing I thought about that was how we are gonna make that clear to user. For instance documenting 20 folders and 100 files, a user might be confused since those folders consist of those files as well, ie I don't know if they're going to understand what we mean by documenting a folder

Reply:
I understand. Maybe this is more of a wording issue. We could say "We will create 120 docs".

Either way, the UI should have this data easily accessible I think. We could change it later though